src/renderers/WebGLRenderer.js

```diff
@@ -1269,6 +1273,11 @@ class WebGLRenderer {
 
 			const currentRenderTarget = _this.getRenderTarget();
 			_this.setRenderTarget( _transmissionRenderTarget );
+
+			_this.getClearColor( _currentClearColor );
+			_currentClearAlpha = _this.getClearAlpha();
+			if ( _currentClearAlpha < 1 ) _this.setClearColor( 0xffffff, 0.5 );
+
```

Comment:
This is a change from my PR. Are you aware that the render target will have premultiplied values?

So the two values of interest in this PR are `0xffffff * 0.5 = 0x7f7f7f` and alpha = `0.5`.

In my PR, the values are `0x7f7f7f * 0.8 = 0x666666` and alpha = `0.8`.

Reply:
Good call, we talked offline. I think we're at consensus now. Correct me if I'm wrong.

Reply:
This is a subjective decision, and can be changed later.